payment-service/src/main/resources/application.properties

```diff
@@ -1,4 +1,12 @@
 spring.application.name=payment-service
 server.port=0
-
 eureka.client.service-url.defaultZone=http://localhost:8761/eureka/
+spring.datasource.url=jdbc:mysql://localhost:3306/libary
+spring.datasource.username=root
+spring.datasource.password=1D1234
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.liquibase.change-log=classpath:db/changelog/db.changelog_master.xml
+spring.jpa.show-sql=true
```

Comment:
it seems to me that spring.jpa.show-sql as false in the main application profile

Reply:
done

Reply:
spring.jpa.show-sql need to be changed to false

Reply:
done